src/app/features/room/RoomView.tsx

```diff
@@ -102,7 +100,7 @@ export function RoomView({ room, eventId }: { room: Room; eventId?: string }) {
   return (
     <Page ref={roomViewRef}>
```

Comment:
Fix: Deduplicate call room headers
```suggestion
    <Page ref={roomViewRef}>
      {!room.isCallRoom() && <RoomViewHeader />}
```